lab_5_scrapper/target_score.txt

```diff
@@ -1 +1 @@
-6
\ No newline at end of file
+10
```

Comment:
are you planning to increase the mark? now you have got 8 according to the spreadsheet

Reply:
I think I won't increase it, should i then change the target score for lab 5?

Reply:
yes. target score for scrapper does not influence target score for pipeline. so reflect your mark (8) in scraper's mark